vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -4,183 +4,184 @@ sidebarDepth: 2
+3.  For `Cluster Service Role`, select the role you created above, `YOUR-EKS-ROLE`. If you choose a different role, it must have ELB permissions so the cluster can create a load balancer in `Networking`. 
+4. Click `Next`.
+5. Use the defaults for `Networking` (Step 2) and click `Next`. 
+6. Use the defaults for `Configure Logging` (Step 3) and click `Next`.
+7. Review your settings and then click `Create`. Cluster provisioning usually takes between 10 and 15 minutes.
+
+See [Creating an Amazon EKS Cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) for more detailed information.
+
+### Add a Node Group to the Cluster
+1. Go to `Services` → `Elastic Kubernetes Service` → `Clusters` and select`YOUR-CLUSTER-NAME` 
+2. Go to the `Compute` tab 
+3. Click `Add Node Group`
+4. Enter the following:
+      * `Name`: Give your group a name, e.g. YOUR-NODE-1
+      * `Node IAM Role`: Select the cluster role you created above. If the role doesn't appear, verify that you added the extra policies to the role.
+      * Click `Next`
+5. For `Subnets` → VPC or Virtual Private Cloud subnets should already be setup and selected. Click `Next`.
+6. Click `Next` again
+7. Review the Compute and Scaling configuration. Typically, the AWS defaults below work:
+     * AMI type: `Amazon Linux 2`
+     * Instance type: `t3.medium`
+Click `Next`
+8. Review the Node Group scaling configuration: 
+      * Set `Maximum size` to `5`. This is over-resourced for a Getting Started instance but offers capacity for adding microservices to your cluster without modifying the Nodegroup. 
+      * Click `Next`
+9. Select `Allow remote access to nodes`. Follow the links to create a new SSH key pair if you don't already have one.
+10. Review your settings and then click `Create`. It may take a few minutes for the node to be created.
+ 
 ### Install the NGINX Ingress Controller
-1. Add the NGINX controller for ingress. This depends on your role having permissions for ELB.
-   - For basic NGINX ingress install run this command
-    ```sh
-    kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.2/deploy/static/provider/cloud/deploy.yaml
-    ```
-   - See <https://kubernetes.github.io/ingress-nginx/deploy/#aws> as well as [this](https://docs.nginx.com/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services/) for more detailed install steps.
-2. Get the ELB external URL for your NGINX install
-   - Run: ```kubectl get services -n ingress-nginx```
-   - Get the value of the external address (EXTERNAL-IP) for the ingress-nginx-controller:
-```
-NAME                                 TYPE           CLUSTER-IP       EXTERNAL-IP                        
-ingress-nginx-controller             LoadBalancer   10.100.102.83    ad234bd11a1ff4dadb44639a6bbf707e-0e0a483d966405ee.elb.us-east-2.amazonaws.com
-```
+Add the NGINX controller for the ingress. This depends on your role having permissions for ELB.
```

Comment:
"depends on" reads as the following clause only applies in certain circumstances; would change to "relies on"